task.go

```diff
@@ -8,7 +8,11 @@ type Task struct {
 	Description  string
 	Command      func(tf *TF)
 	Dependencies Deps
+	Parameters   Params
```

Comment:
What do you think of renaming `Parameters` to `Params` for consistency?

* probably I would rename `Dependencies` to `Deps` as well

Reply:
I will skip it for now. I have an idea of how to redesign the parameter registration. I want to make a draft PR for it first.

Reply:
Refactored 😄